### PR TITLE
When installing latest Knative, show what it is

### PR DESCRIPTION
**Pull request by adrcunha**
Also factor out the common address of the YAML files, so they can all be updated with a single change.

### CODE REVIEW

File: scripts/library.sh [hunk 1]
```diff
@@ -351,11 +352,14 @@ function report_go_test() {
...
   header "Starting Knative Serving"
   subheader "Installing Istio"
+  echo "Installing Istio CRD from ${KNATIVE_ISTIO_CRD_YAML}"
   kubectl apply -f ${KNATIVE_ISTIO_CRD_YAML} || return 1
+  echo "Installing Istio from ${KNATIVE_ISTIO_YAML}"
   kubectl apply -f ${KNATIVE_ISTIO_YAML} || return 1
   wait_until_pods_running istio-system || return 1
   kubectl label namespace default istio-injection=enabled || return 1
   subheader "Installing Knative Serving"
+  echo "Installing Serving from ${KNATIVE_SERVING_RELEASE}"
```
Comment (chaodaiG) [inline scripts/library.sh]: nit-picking: just for consistency between subheader and this message:
```suggestion
   echo "Installing Knative Serving from ${KNATIVE_SERVING_RELEASE}"
```

**Comment by chaodaiG** [comment]
/lgtm

just a nit-picking comment on message

File: scripts/library.sh (hunk 1 above)
Comment (adrcunha) [reply scripts/library.sh]: You LGTMed but did not hold it, so the PR was merged without your suggestion. :/

Comment (chaodaiG) [reply scripts/library.sh]: yeah it's no big deal, I don't think it should be a PR blocker. We can add it in the future

**Pull request merged**